easyrest/auth.py

```diff
@@ -0,0 +1,92 @@
+"""Module for security implementation. Contains functions to add, remove token
+(remember, forget), decorator to check access, function to asign token model 
+to request. 
+"""
+
+from passlib.pwd import genword
```

Comment:
PEP8: three groups of imports. Built-ins first, then frameworks and other dependencies, last your modules.